scripts/deploymentUtils.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+const resolve = require('path').resolve;
+
+require('dotenv').config({ path: resolve('.env') });
+
```

Comment:
why is there new line after every import?

Reply:
Weird styling, fixing.